scripts/generate.sh

```diff
@@ -67,3 +67,5 @@ mv -f src/README.md ./README.md
 mv -f src/composer.json composer.json
 find ./docs -type f ! -name "DecryptionKey.md" ! -name "Sealed.md" -exec rm {} +
 mv -f src/docs/* ./docs
+
+docker run --rm -v $(pwd):/code ghcr.io/php-cs-fixer/php-cs-fixer:${FIXER_VERSION:-3-php8.3} fix src
```

Comment:
As I see it didn't fix formatting in `src/Model/BotdResult.php` for example.
Maybe we will need to extend default rules.